### PR TITLE
Bring back single level side nav

### PR DESCRIPTION
**Pull request by cindylovescoding**
Change new side nav to the original side nav with a few changes:
1. Added "overview" menu and "full detector list" menu:

image

2. Put "analysis" under "detectors" with other categories:

image

### CODE REVIEW

File: AngularApp/.npmrc [hunk 1]
```diff
@@ -1,3 +0,0 @@
-registry=https://pkgs.dev.azure.com/app-service-diagnostics-portal/_packaging/app-services-diagnostics-portal-nuget-feed/npm/registry/
```
Comment (ShekharGupta1988) [inline AngularApp/.npmrc:1]: Looks like deletion of this file is causing angular build to fail:

https://dev.azure.com/app-service-diagnostics-portal/app-service-diagnostics-portal/_build/results?buildId=9201&view=logs&j=6b902995-b73d-5f5c-66fd-a7f66c857d2c&t=1a078ffa-9e50-5a32-4010-fb8aa33da309
##[error]Error: The .npmrc file you selected at /home/vsts/work/1/s/AngularApp/.npmrc does not currently exist.

**Comment by ShekharGupta1988** [comment]
Great work @cindylovescoding.
Couple of things to consider:
- Possible to include 'All Detectors' inside the Detectors collapsible item ? (and not have the Full Detectors List item) 
- Can you rearrange the side items to :
        - Overview
        - Detectors
        - Develop
        - Gists
        - Configuration

**Review by ShekharGupta1988 — APPROVED** [review]

File: AngularApp/.npmrc (hunk 1 above)
Comment (cindylovescoding) [reply AngularApp/.npmrc:1]: > Great work @cindylovescoding. Couple of things to consider:
> 
> * Possible to include 'All Detectors' inside the Detectors collapsible item ? (and not have the Full Detectors List item)
> * Can you rearrange the side items to :
>   - Overview
>   - Detectors
>   - Develop
>   - Gists
>   - Configuration

Sounds good Shekhar. I will make the change

Comment (cindylovescoding) [reply AngularApp/.npmrc:1]: > Looks like deletion of this file is causing angular build to fail:
> 
> https://dev.azure.com/app-service-diagnostics-portal/app-service-diagnostics-portal/_build/results?buildId=9201&view=logs&j=6b902995-b73d-5f5c-66fd-a7f66c857d2c&t=1a078ffa-9e50-5a32-4010-fb8aa33da309 ##[error]Error: The .npmrc file you selected at /home/vsts/work/1/s/AngularApp/.npmrc does not currently exist.

This file was causing local authentication error to Devops Nuget package feeds but looks like it is still used by the build pipeline. I will add this back

**Pull request merged**